doc/users/next_whats_new/subplots_adjust.rst

```diff
@@ -0,0 +1,9 @@
+Resetting the subplot parameters for figure.clear()
+---------------------------------------------------
+
+When calling `.Figure.clear()` the settings for `.gridspec.SubplotParams` are restored to the default values.
+
+The `.gridspec.SubplotParams` object has a new method to get the subplot parameters :meth:`~.SubplotParams.to_dict` and a
+method to reset the parameters to the defaults :meth:`~.SubplotParams.reset`
+
+(contributed by @eendebakpt based on work by @fredrik-1)
```

Comment:
We don't have "(contributed by..)" in the what's new

```suggestion
```

Reply:
If you want to acknowledge someone, add their GitHub email to the commit message (which I admit is hard to find sometimes).